haas/cli.py

```diff
@@ -444,6 +474,27 @@ def stop_console(node):
     do_delete(url)
 
 @cmd
+def create_admin_user(username, password):
```

Comment:
It's described here:

https://github.com/CCI-MOC/haas/pull/541/files#diff-26dee18463b5da190f343abcc74cf708R284